ex1/functions.py

```diff
@@ -90,26 +261,149 @@ def f():
 # Define a function named args_to_dict that takes three arguments and returns a
 # dictionary with the position of the argument as the key (starting at 0) and
 # the argument as the value.
+def args_to_dict(a,b,c):
+	"""
+    Writes three arguments to dictionary
+
+    Parameters
+    ----------
+    a: object
+        First object to add to dictionary
+    b: object
+        Second object to add to dictionary
+	c: object
+		Third object to add to dictionary
+	
+    Returns
+    -------
+	dictionary
+        Returns a dictionary with keys 0 to 2 and given values a,b,c
+    """
+	return {0:a,1:b,2:c}
 
 # BONUS: Write a function named args_to_dict_general that does the same for any
 # number of arguments
+def args_to_dict_general(*args):
+	"""
+    Writes three arguments to dictionary
 
+    Parameters
+    ----------
+    *args: objects
+		Given objects to add to dictionary
+	
+    Returns
+    -------
+	d: dictionary
+        Returns a dictionary with keys 0 to n and given values n
+    """
+	d = {}
+	i = 0
+	for arg in args:
+		d[i] = arg
+		i+=1
+	return d
```

Comment:
This can be solved using `enumerate` instead of the manual increment of the `i` variable.